libs/net/dal/Migrations/Initial/Up/PostUp/01-DataSource.sql

```diff
@@ -2932,7 +2932,7 @@ INSERT INTO public.data_source (
   , CBC_CAPTURE_CODE
   , ''
   , true -- is_enabled
-  , NULL -- content_type_id
+  , 3 -- content_type_id
```

Comment:
This is incorrect.  CBC capture does not generate any content, which is `0`.  It does highlight that I should probably update the DB entity so that it's not allowed to be `null` though.  As all data sources that don't generate content should be set to `0`.

Reply:
fixed!